package.json

```
@@ -4,8 +4,8 @@
   "private": false,
   "version": "0.0.1-d",
   "description": "A package to parse API3 operations metadata",
-  "main": "./dist/index",
-  "types": "./dist/validation/types",
+  "main": "./dist/index.js",
+  "types": "./dist/index.js",
```

Shouldn't this be like this?
```suggestion
  "types": "./dist/index.d.ts",
```
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

---

ah interesting, I was wondering, thanks :)